src/customer-ontology.ttl

```diff
@@ -0,0 +1,91 @@
+@prefix : <http://bibsysdev.github.io/src/customer-ontology#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <https://schema.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+# Customer-related concepts
+
+:Customer a rdf:Class ;
+  rdfs:comment "Customer institution in NVA" .
+
+:VocabularySetting a rdf:Class ;
+  rdfs:comment "Setting for a controlled vocabulary for a customer institution" .
+
+:VocabularyStatus a rdf:Class ;
+  rdfs:comment "Status for a controlled vocabulary for a customer institution" .
+
+:name a rdf:Property ;
+  rdfs:comment "The name of the customer institution" ;
```

Comment:
The descriptions of the properties are quite short and they basically rephrase the the name of the property. In the cases where the explanation cannot be more detailed, can we add an example value?

Reply:
It's typically the case that a vocabulary includes an rdfs:label, which repeats the token (so here, rdf:label "name"). For the comment, it's hard to find an alternative for name, but I'd suggest, e.g. "A string representing what the institution is called"